test/over_react/component/error_boundary_prod_test.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright 2022 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('browser')
+library error_boundary_test;
+
+import 'package:meta/meta.dart';
+import 'package:over_react/over_react.dart' hide ErrorBoundary;
+import 'package:react_testing_library/react_testing_library.dart' as rtl;
+import 'package:over_react/components.dart';
+import 'package:test/test.dart';
+
+part 'error_boundary_prod_test.over_react.g.dart';
+
+void main() {
+  group('ErrorBoundary', () {
```

Comment:
If it's not too much effort, could we also run these in the non-prod test suites? Not super important, but since we already have the tests written, it'd be nice to have an automated check that ensures behavior is consistent across prod and non-prod bundles.